test/Microsoft.AspNetCore.Authentication.Test/Cookies/CookieMiddlewareTests.cs

```diff
@@ -851,68 +740,85 @@ private Task SignOutAsWrong(HttpContext context)
         }
 
         [Fact]
-        public async Task MapWillNotAffectChallenge()
+        public async Task MapWillAffectChallenge()
```

Comment:
Note: this is a behavior change from old stack to new stack (Map does affect redirect url for challenge now that its not tied to any middleware)